core/src/main/groovy/com/novoda/gradle/release/BintrayConfiguration.groovy

```diff
@@ -16,16 +17,17 @@ class BintrayConfiguration {
 
         PropertyFinder propertyFinder = new PropertyFinder(project, extension)
 
-        project.bintray {
+        def bintrayExtension = project.extensions.findByName("bintray") as BintrayExtension
+        bintrayExtension.with {
             user = propertyFinder.bintrayUser
             key = propertyFinder.bintrayKey
             publish = extension.autoPublish
             dryRun = propertyFinder.dryRun
             override = propertyFinder.override
 
-            publications = extension.publications ?: project.plugins.hasPlugin('com.android.library') ? ['release'] : [ 'maven' ]
+            publications = extension.publications ?: project.plugins.hasPlugin('com.android.library') ? ['release'] : ['maven']
 
-            pkg {
+            pkg.with {
```

Comment:
are you sure this works at all? `pkg` is a method that receives a `Closure` as parameter, not an Object

Reply:
We don't call the method here. 
We setup the pkg property..

Reply:
Well, not entirely true: the code above can be rewritten as [`pkg({ ... })`](https://github.com/bintray/gradle-bintray-plugin/blob/master/src/main/groovy/com/jfrog/bintray/gradle/BintrayExtension.groovy#L34-L36), but given the method receives the closure as last parameter you can just omit the `()`. 

It's also true that there is a public [`pkg`](https://github.com/bintray/gradle-bintray-plugin/blob/master/src/main/groovy/com/jfrog/bintray/gradle/BintrayExtension.groovy#L16) property available from `BintrayExtension`, but I reckon that is not the one we should be using, as it looks more an oversight on their end.